src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -158,4 +158,22 @@ public function getFlashes(string|array $types = null): array
 
         return $result;
     }
+
+    public function getCurrentRoute(): ?string
+    {
+        if (!isset($this->requestStack)) {
+            throw new \RuntimeException('The "app.currentRoute" variable is not available.');
```

Comment:
I think I would prefer having a variable named `app.current_route`

Reply:
You're right, done and I've update the PR description as well. Thanks!

Reply:
We need to change the method names too, don't we? And update the changelog as well.

Reply:
Indeed. I got confused as I thought I remembered snake case in method names was converted to camel case by Twig.
All fixed now, thanks again!